iree/compiler/Conversion/HLOToLinalg/FusionOfTensorOps.cpp

```diff
@@ -89,7 +89,22 @@ struct FusionOfTensorOpsPass
     // indexed generic operations. If an operation is used in a named op, it
     // will be computed anyway, so the consumers can just use that value.
     linalg::ControlElementwiseOpsFusionFn controlFn =
-        [](const OpResult &producer, const OpOperand & /*consumer*/) {
+        [](const OpResult &producer, const OpOperand &consumer) {
+          // TODO(ataei): Enable fusion with reduction consumer. Currently
```

Comment:
Just use #5045 for the TODO here. Ill update the bug with context of things left to do there.